code/__DEFINES/atmospherics.dm

```diff
@@ -153,6 +153,7 @@
 
 //OPEN TURF ATMOS
 #define OPENTURF_DEFAULT_ATMOS		"o2=22;n2=82;TEMP=293.15" //the default air mix that open turfs spawn
+#define OPENTURF_LOW_PRESSURE		"o2=14;n2=30;;TEMP=293.15"

```

Comment:
Looks like you got an extra semicolon here?

Reply:
Woops. Fixed.